test/js/web/fetch/fetch.stream.test.ts

```diff
@@ -213,7 +213,12 @@ describe("fetch() with streaming", () => {
           .listen(0);
 
         const address = server.address() as AddressInfo;
-        const url = `http://${address.address}:${address.port}`;
+        let url;
+        if (address.family == "IPv4") {
+          url = `http://${address.address}:${address.port}`;
+        } else {
+          url = `http://[${address.address}]:${address.port}`;
```

Comment:
IPv6 need to be enclosed in brackets. The previous tests passed because `localhost` was returned.